File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12015_:::_Google_is_Feeling_Lucky/Main.c
#include "stdio.h"

int main()
{
	int casos, max, V[10+1], c, i;
	char urls [10+1][100+10];
	scanf("%d", &casos);
	
	for(c=1; c<=casos; c++)
	{
		max=0;
		for(i=0; i<10; i++)
		{
			scanf("%s %d", urls[i], &V[i]);
			if(V[i]>max) max=V[i];
		}
		
		printf("Case #%d:\n", c);
		for(i=0; i<10; i++)
			if(V[i]==max)
				puts(urls[i]);
	}

	return 0;
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/12015_:::_Google_is_Feeling_Lucky/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine()), max, V[]=new int[10];
		String[] urls=new String[10], aux;
		
		for(int c=1; c<=casos; c++)
		{
			max=0;
			for(int i=0; i<10; i++)
			{
				aux=br.readLine().split("\\s+");
				urls[i]=aux[0];
				V[i]=Integer.parseInt(aux[1]);
				if(V[i]>max) max=V[i];
			}
			
			System.out.println("Case #"+c+":");
			for(int i=0; i<10; i++)
				if(V[i]==max)
					System.out.println(urls[i]);
		}
		//-----------------------------------------------------------
		br.close();
	}
}
